src/controller/addonController.ts

```diff
@@ -1,6 +1,7 @@
 import * as extract from "extract-zip";
 import * as fs from "fs";
 import fetch from "node-fetch";
+import path = require("path");
```

Comment:
Try not to mix import vs. require statements. This should work:

```suggestion
import * as path from "path";
```